README.md

```diff
@@ -27,6 +27,36 @@ Installers
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+#### Debian Repository
+
+- Save the following as `/etc/apt/sources.list.d/etcher.list`:
+
+```
+deb https://dl.bintray.com/resin-io/debian stable etcher
+```
+
+- Trust Bintray.com's GPG key:
+
+```
```

Comment:
I guess this (and the code-line below) could also be tagged as `sh` format, as the brew line is?